index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section>
+    <nav class="navbar">
+        <div class="container">
```

Comment:
- [Optional] Please remove this weird looking border here so your code matches the design
![image](https://user-images.githubusercontent.com/74744787/212504642-3beacefd-3981-4062-a216-67f2518b22cd.png)